### ISSUE TITLE
Fatal error when building

### ISSUE THREAD

**Issue by junapoe** [AhoyISki/wayfire-firedecor#3]
Hi, I tried compiling and installing but it seems to not build. I don't know if I'm doing anything wrong so sorry if I'm missing something, lol

Here is the output when I run  `meson compile -C build`

````[1/6] Compiling C++ object src/libfiredecor.so.p/firedecor-subsurface.cpp.o
FAILED: src/libfiredecor.so.p/firedecor-subsurface.cpp.o
c++ -Isrc/libfiredecor.so.p -Isrc -I../src -I/usr/include/libevdev-1.0 -I/usr/include/libxml2 -I/usr/include/libdrm -I/usr/include/pixman-1 -I/usr/include/librsvg-2.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/sysprof-4 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/cairo -I/usr/include/lzo -I/usr/include/freetype2 -I/usr/include/harfbuzz -I/usr/include/pango-1.0 -I/usr/include/fribidi -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -std=c++17 -O0 -g -DWLR_USE_UNSTABLE -DWAYFIRE_PLUGIN -fPIC -pthread -MD -MQ src/libfiredecor.so.p/firedecor-subsurface.cpp.o -MF src/libfiredecor.so.p/firedecor-subsurface.cpp.o.d -o src/libfiredecor.so.p/firedecor-subsurface.cpp.o -c ../src/firedecor-subsurface.cpp
../src/firedecor-subsurface.cpp:17:10: fatal error: cairo-simpler.hpp: No such file or directory
   17 | #include "cairo-simpler.hpp"
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
[5/6] Compiling C++ object src/libfiredecor.so.p/firedecor-theme.cpp.o
In file included from ../src/firedecor-theme.cpp:13:
/usr/include/librsvg-2.0/librsvg/rsvg.h:125:66: warning: "/*" within comment [-Wcomment]
  125 |  * included through XML entities) if those files are in `/foo/bar/*` or in `/foo/bar/*\/.../*`.
      |
/usr/include/librsvg-2.0/librsvg/rsvg.h:125:85: warning: "/*" within comment [-Wcomment]
/usr/include/librsvg-2.0/librsvg/rsvg.h:125:92: warning: "/*" within comment [-Wcomment]
ninja: build stopped: subcommand failed.```

**Comment by UltraBlackLinux** [comment]
I am facing the same issue, sadly.

**Comment by Cognomines** [comment]
I have tried looking for this mysterious cairo-simpler.hpp header file
. I cannot find it in I have tried commenting out #include cairo-simpler.hpp and uncommenting cairo.h in firdecor-subsurface.cpp. It does not seem to help much. I run into issues with wf::color_t and wf::point_t. 
[typescript.log](https://github.com/AhoyISki/wayfire-firedecor/files/8558267/typescript.log)

See log generated by /usr/bin/script from util-linux
```bash
$ curl -LJ0 https://github.com/AhoyISki/wayfire-firedecor/files/8558267/typescript.log | less -r
```
